llvm/include/llvm/BinaryFormat/ELF.h

```diff
@@ -1756,6 +1752,26 @@ enum : unsigned {
   GNU_PROPERTY_AARCH64_FEATURE_1_GCS = 1 << 2,
 };
 
+// aarch64 PAuth platforms.
+enum : unsigned {
+  AARCH64_PAUTH_PLATFORM_INVALID = 0x0,
+  AARCH64_PAUTH_PLATFORM_BAREMETAL = 0x1,
+  AARCH64_PAUTH_PLATFORM_LLVM_LINUX = 0x10000002,
+};
+
+// Bit positions of version flags for AARCH64_PAUTH_PLATFORM_LLVM_LINUX.
+enum : unsigned {
+  AARCH64_PAUTH_PLATFORM_LLVM_LINUX_VERSION_INTRINSICS = 0,
```

Comment:
Where are these constants defined?

Reply:
Invalid and baremetal platform values are reserved in https://github.com/ARM-software/abi-aa/blob/main/pauthabielf64/pauthabielf64.rst (constant names themselves are not defined there though). Regarding everything related to llvm_linux platform - it's not defined anywhere and I've defined it here since we need some sort of working ELF marking scheme to at least test things. If different constant names and/or values are preferred or there is some specification which needs to be updated correspondingly - please let me know.